app/controllers/api/v2/rh_cloud/inventory_controller.rb

```diff
@@ -28,6 +28,15 @@ def generate_report
           }, status: :ok
         end
 
+        api :GET, "/organizations/:organization_id/rh_cloud/missing_hosts", N_("Grab hosts that are missing in RH Cloud")
```

Comment:
Unlike Katello, we have separated UI and API requests. I would suggest a new controller `MissingHostsController.rb` under the [`foreman_inventory_upload`](https://github.com/theforeman/foreman_rh_cloud/tree/foreman_3_7/app/controllers/foreman_inventory_upload) folder.

Reply:
Is the new controller in addition to the new route I added here, or are you saying get rid of this one?

I'm guessing new one for the ui and this one stay for the api.